solutions/usaco-1088.mdx

```diff
@@ -11,3 +11,151 @@ author: Melody Yu (Video), Tyler Xiao (Video)
 
 Note: The video solution might not be the same as other solutions. Code in Java.
 <Youtube id="7eMHxpDQJEw" />
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N^2)$
+
+<LanguageSection>
+
+<CPPSection>
+
+```cpp
+#include <iostream>
+#include <algorithm>
+using namespace std;
+
+int main()
+{
```

Comment:
dear god can you not put curly braces on their own lines

Reply:
...
> why? i think this is fine...

Reply:
consistency, like 80% of the other solutions have curly braces on the same line

Reply:
i think this is just coding style though... code only needs to be consistent with itself, not with other solutions...

>  Of course, we still strive for code that is readable and understandable.

> Please read the following resource before contributing code (although you do not have to follow all of the recommendations):

*https://usaco.guide/general/adding-solution?lang=cpp#code-conventions*

I think my code is still readable and understandable, and it says that I don't have to follow all the recommendations

Reply:
yeah but it's just generally better to be consistent w/ other sols
you don't see people using snake_case for java sols or camelCase for python sols